realtime/socketIOEmitter.js

```diff
@@ -27,7 +27,14 @@ module.exports = (io, key, mssgObj) => {
   }
   for (const nsp in io.nsps) {
     if (nsp && rtUtils.shouldIEmitThisObj(nsp, obj)) {
-      io.of(nsp).emit(key, JSON.stringify(mssgObj));
+      // obj contains only the newest values
+      io.of(nsp).emit(key, JSON.stringify({
+        key: {
+          new: {
+            obj,
+          },
+        },
+      }));
     }
```

Comment:
@annyhe Can you make sure that you can still see the socketio events come into the browser after these changes?